backend/inspirehep/sitemap/templates/sitemap/index.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I don't think templating XML is a great approach. It might work here as it's rather simple, but in general it's better to use something like `lxml.etree.ElementMaker` to build the element tree (we do this in https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/orcid/builder.py).

Reply:
This is actually copy-paste from zenodo.

yeah I see, but that looks like so much hustle for something simple like sitemap.

Reply:
I added XML (syntax only) validation during tests.